<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="authenticateGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>431</width>
      <height>681</height>
     </rect>
    </property>
    <property name="title">
     <string>Authenticate</string>
    </property>
    <widget class="QPushButton" name="authenticateBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Authenticate</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="authenticateConsoleOutput">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>381</width>
       <height>501</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="downloadGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>40</y>
      <width>621</width>
      <height>681</height>
     </rect>
    </property>
    <property name="title">
     <string>Download</string>
    </property>
    <widget class="QRadioButton" name="landsatRBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Landsat</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="sent2RBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sentinel-2</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>561</width>
       <height>501</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="landsatPage">
      <widget class="QLabel" name="goeJsonLbl">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Input GeoJSON:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="geoJsonPath">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>341</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="geoJsonPathBtn">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>100</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="startDateLbl">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>160</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Date:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="startDateSelector">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>160</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="endDateLbl">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>End Date:</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="endDateSelector">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="cloudCoverLbl">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Cloud Cover:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cloudCoverTxt">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>270</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>0-100</string>
       </property>
      </widget>
      <widget class="QPushButton" name="downloadBtn">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>400</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAuthenticate">
   <property name="text">
    <string>Authenticate</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>landsatRBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>cloudCoverLbl</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>landsatRBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>cloudCoverTxt</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sent2RBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>cloudCoverLbl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sent2RBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>cloudCoverTxt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
